src/main/java/com/github/paweladamski/httpclientmock/HttpClientMockBuilder.java

```diff
@@ -1,333 +1,333 @@
-
-  /**
-   * Adds action which returns empty message and provided status.
-   *
-   * @param statusCode status to return
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doReturnStatus(int statusCode) {
-    return responseBuilder.doReturnStatus(statusCode);
-  }
-
-  /**
-   * Adds action which throws provided exception.
-   *
-   * @param exception exception to be thrown
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doThrowException(IOException exception) {
-    return responseBuilder.doThrowException(exception);
-  }
-
-  /**
-   * Adds action which returns provided JSON in UTF-8 and status 200. Additionally it sets "Content-type" header to "application/json".
-   *
-   * @param response JSON to return
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doReturnJSON(String response) {
-    return responseBuilder.doReturnJSON(response);
-  }
-
-  /**
-   * Adds action which returns provided JSON in provided encoding and status 200. Additionally it sets "Content-type" header to "application/json".
-   *
-   * @param response JSON to return
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doReturnJSON(String response, Charset charset) {
-    return responseBuilder.doReturnJSON(response, charset);
-  }
-
-  /**
-   * Adds action which returns provided XML in UTF-8 and status 200. Additionally it sets "Content-type" header to "application/xml".
-   *
-   * @param response JSON to return
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doReturnXML(String response) {
-    return responseBuilder.doReturnXML(response);
-  }
-
-  /**
-   * Adds action which returns provided XML in UTF-8 and status 200. Additionally it sets "Content-type" header to "application/xml".
-   *
-   * @param response JSON to return
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doReturnXML(String response, Charset charset) {
-    return responseBuilder.doReturnXML(response, charset);
-  }
-
-  /**
-   * Adds action which returns provided URL-encoded parameter response in UTF-8 and status 200. Additionally it sets "Content-type" header to "application/x-www-form-urlencoded".
-   *
-   * @param parameters parameters to return
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doReturnFormParams(Collection<NameValuePair> parameters) {
-    return doReturnFormParams(parameters, StandardCharsets.UTF_8);
-  }
-
-  /**
-   * Adds action which returns provided URL-encoded parameter response with status 200. Additionally it sets "Content-type" header to "application/x-www-form-urlencoded".
-   *
-   * @param parameters parameters to return
-   * @return response builder
-   */
-  public HttpClientResponseBuilder doReturnFormParams(Collection<NameValuePair> parameters, Charset charset) {
-    return responseBuilder.doReturnFormParams(parameters, charset);
-  }
-}
+package com.github.paweladamski.httpclientmock;

```

Comment:
Can you tell me what has been changed in this file?

Reply:
Some of the information in the Javadocs was wrong with respect to character encoding, and some of the wording was inconsistent.

Line 285
Old: `Adds action which returns provided JSON in provided encoding and status 200. Additionally it sets "Content-type" header to "application/json".`
New: `Adds action which returns provided JSON in provided charset and status 200. Additionally it sets "Content-type" header to "application/json".`

Line 305
Old: `Adds action which returns provided XML in UTF-8 and status 200. Additionally it sets "Content-type" header to "application/xml".`
New: `Adds action which returns provided XML in provided charset and status 200. Additionally it sets "Content-type" header to "application/xml".`

Line 325
Old: `Adds action which returns provided URL-encoded parameter response with status 200. Additionally it sets "Content-type" header to "application/x-www-form-urlencoded".`
New: `Adds action which returns provided URL-encoded parameter response in provided charset and status 200. Additionally it sets "Content-type" header to "application/x-www-form-urlencoded".`

Reply:
I was rather asking why the whole file has changed? Have you changed encoding or line endings?

Reply:
My git configuration is a little wonky. I am using Windows and git hasn't been doing the line ending conversions when I clone a repo. So, before I commit anything, I have been manually telling Eclipse to convert the line endings to Unix-style. I may have forgotten to do that with this file.

Reply:
Here is a better view of the changes: https://github.com/PawelAdamski/HttpClientMock/pull/22/commits/9e0dcefa4a150da313990de682af928be3399033